docker-compose.yml

```diff
@@ -14,6 +14,7 @@ services:
       - postgis
 
   postgis:
+    shm_size: 2g
```

Comment:
this fixes some random errors i occasionally get when running a complex query locally 🤷